packages/client/src/debug/DebugDecisionTable.tsx

```diff
@@ -37,21 +38,53 @@ const DebugDecisionTable: FunctionComponent = () => {
   const autofilled = allQuestions.filter((q: Question) => q.autofill);
 
   return (
-    <HiddenDebugInfo>
-      <Accordion title="Checker debug-informatie:">
-        <div style={{ display: "block" }}>
-          {autofilled.filter((q: Question) => q.autofill).length > 0 && (
-            <>
-              <h1>Autofill</h1>
-              {autofilled.map((q: Question) => (
-                <p key={q.id}>
-                  <QuestionSummary question={q} />: <Answer question={q} />
-                </p>
-              ))}
-            </>
-          )}
+    <HideForPrint>
```

Comment:
This file is only ordered differently and has `HideForPrint` as wrapper, that's it